src/pages/updates.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
are we sure this is the correct name for this page?

Reply:
I already opened an other PR where I fixed this